schema/deploy/tables/ciip_application_wizard_001.sql

```diff
@@ -0,0 +1,12 @@
+-- Deploy ggircs-portal:tables/ciip_application_wizard_001 to pg
+-- requires: tables/ciip_application_wizard
+
+begin;
+
+alter table ggircs_portal.ciip_application_wizard add column is_active boolean default true;
+drop index ggircs_portal.ciip_application_wizard_form_position_uindex;
+create index ciip_application_wizard_form_position_uindex on ggircs_portal.ciip_application_wizard(form_position);
```

Comment:
If it's not a unique index, then the name shouldn't have `uindex` in it. Not sure the index it needed, given that this table will have very few records
```suggestion
create index ciip_application_wizard_form_position_index on ggircs_portal.ciip_application_wizard(form_position);
```